src/transformers/models/t5/modeling_flax_t5.py

```diff
@@ -732,8 +732,13 @@ def __call__(
         return_dict: bool = True,
         deterministic: bool = True,
         init_cache: bool = False,
+        input_embeds=None
     ):
-        hidden_states = self.embed_tokens(input_ids)
+        if input_embeds is not None:
```

Comment:
(nit) Could we write this `if`/`else` statement in the reverse order to which it currently is? Would improve the clarity!